src/components/MissingSkuModal.vue

```diff
@@ -41,8 +41,12 @@
         </ion-list>
 
         <ion-list v-if="segmentSelected === 'completed'">
-          <ion-item v-for="item in getCompletedItems()" :key="item.shopifyProductSKU">
+          <ion-item v-for="item in getCompletedItems()" :key="item.shopifyProductSKU" lines="full">
+            <ion-thumbnail slot="start">
+              <Image :src="item.imageUrl" />
+            </ion-thumbnail>
             <ion-label>
+              <p>{{ item.parentProductName }}</p>
```

Comment:
Use class="overline" whenever placing content about the primary piece of info in an ion-item